relays.go

```diff
@@ -176,6 +176,20 @@ func (r *Relays) Register(
 	return relay
 }
 
+func (r *Relays) Delete(identifier string) {
+	uri := fmt.Sprintf("/relay/%s", identifier)
```

Comment:
The relay name is an arbitrary string accepted from the user. You want to use `url.PathEscape` on that identifier.

Reply:
note for later, when you have tests: test for names with quotes, url-escapable characters, unicode. I recommend :pile_of_poo: as a good character to test with.

Reply:
Also whitespace, vertical tabs. On a serious note, make sure korean is accepted in all forms. See the go concepts of 'runes'.

Reply:
Fixed in the lastest force-push.